src/logic/operate.js

```diff
@@ -13,13 +13,17 @@ const operate = (numberOne, numberTwo, operation) => {
       results = big(numberOne).times(numberTwo).toString();
       break;
     case '÷':
-      results = numberTwo === 0 ? 'undefined' : big(numberOne).div(numberTwo).toString();
+      results = numberTwo === '0' ? 'undefined' : big(numberOne).div(numberTwo).toString();
```

Comment:
Let's either handle division by 0 differently or alternatively handle invalid numbers from been passed into the `big` function. The demo below shows how the app crash when trying to operate values that are not numbers.

![cal-error](https://user-images.githubusercontent.com/36057474/71616822-767b7280-2bb8-11ea-8f67-350744efe83f.gif)